ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/editTrustPage.js

```diff
@@ -0,0 +1,25 @@
+class EditTrustPage {
```

Comment:
Not sure if this should be called 'EditTerritoryPage' but not blocking the pr for this

Reply:
This has been changed :)